install_config/configuring_local.adoc

```diff
@@ -0,0 +1,139 @@
+[[install-config-configuring-local]]
+= Configuring for Local Volume
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+{product-title} can be configured to access
+xref:../install_config/persistent_storage/persistent_storage_local.adoc#install-config-persistent-storage-persistent-storage-local[Local
+Volumes] for application data.
+
+The local volume provisioner allows using local storage within {product-title} and supports:
+
+* Volumes, and
+* Persistent Volumes
+
+[NOTE]
+====
+Local volume is an alpha feature and may change in a future release of {product-title}.
+====
+
+[[local-volume-starting-cluster-with-local-disks]]
+=== Starting a Cluster with Local Disks
+[[local-volume-config-baremetal-environments]]
+==== Bare metal Environments
+. Partition and format the disks on each node (according to the application's requirements).
+. Mount all the filesystems under one discovery directory per `StorageClass`. The directories are specified in a *_configmap_*. By default, the discovery directory is *_/mnt/disks_* and storage class is `local-storage`.
+. Configure the API Server, controller-manager, scheduler, and all kubelets with the `PersistentLocalVolumes` feature gate.
+.. Edit *_master-config.yaml_*
++
+[source, yaml]
+----
+apiServerArguments:
+   feature-gates:
+   - PersistentLocalVolumes=true
+ controllerArguments:
+   feature-gates:
+   - PersistentLocalVolumes=true
+----
+
+.. Edit the *_node-config.yaml_* on all nodes.
++
+[source, yaml]
+----
+kubeletArguments:
+   feature-gates:
+     - PersistentLocalVolumes=true
+----
+
+[[local-volume-config-test-clusters]]
+==== Local Test Cluster
```

Comment:
Please remove this whole chapter, it is Kubernetes specific and does not work on OpenShift. Namely, there is no hack/local-up-cluster.sh in OpenShift

Reply:
Thank you @jsafrane I'll update this in the next iteration.